<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_browsefile">
      <property name="text">
       <string>Browse XML Files</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_check">
      <property name="text">
       <string>Check File Consistency</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_format">
      <property name="text">
       <string>Format XML File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_converttojson">
      <property name="text">
       <string>Convert to Json</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_compress">
      <property name="text">
       <string>compress</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_minify">
      <property name="text">
       <string>Minify XML File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
